.codegen/service.go.tmpl

```diff
@@ -45,17 +51,36 @@ func New() *cobra.Command {
 		Short: `{{.Summary | without "`"}}`,
 		Long: `{{.Comment  "  " 80 | without "`"}}`,
 		{{- end }}
+		{{- if not .HasParent }}
```

Comment:
What do the GroupID and Annotations are for, and why are not needed for nested services?

Reply:
They are used to group commands by their service in the help output.